src/main/java/org/jabref/gui/exporter/ExportCommand.java

```diff
@@ -58,18 +57,6 @@ private void export(Path file, FileChooser.ExtensionFilter selectedExtensionFilt
             FileUtil.addExtension(file, selectedExtension);
         }
 
-        if (Files.exists(file)) {
-            // Warn that the file exists:
-
-            boolean overwriteFilePressed = frame.getDialogService().showConfirmationDialogAndWait(Localization.lang("Export"),
```

Comment:
I am just curious: Why is this removed?

Reply:
https://github.com/JabRef/jabref/commit/dfd62b0fbff053da8c76e2e9d5ca38a46ea9daca#diff-dbfbf1695502f56fd92d434d1dc1d641